src/tribol/interface/tribol.cpp

```diff
@@ -608,13 +671,13 @@ int getJacobianCSRMatrix( int** I, int** J, real** vals, int csId,
       }
       case SINGLE_MORTAR:
       {
-         SLIC_WARNING("tribol::getCSRMatrix(): CSR format not currently implemented with "
+         SLIC_WARNING("tribol::getJacobianCSRMatrix(): CSR format not currently implemented with "
```

Comment:
Should these be `SLIC_WARNING_ROOT` to avoid excessive messages?

Reply:
I'm going to leave this for now. I don't know if ranks with null-meshes will always call this routine. Sort of depends on how the host-code implements it. I think more implicit integration/testing will shed light on this.